java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -1151,8 +1140,8 @@ private void trainInfoToDialog(TrainInfo info) {
 
     private TrainInfo dialogToTrainInfo() {
         TrainInfo info = new TrainInfo();
-        info.setTransitName((String) transitSelectBox.getSelectedItem());
-        info.setTransitId(selectedTransit.getDisplayName());
+        info.setTransitName(transitSelectBox.getSelectedItem().getDisplayName());
+        info.setTransitId(transitSelectBox.getSelectedItem().getDisplayName());
```

Comment:
Is the `transitSelectBox` combo box known to always have items? If not, this might give a NullPointerException.

Reply:
This can only happen if the transits are deleted after starting dispatcher, as Dispatcher wont start without at least one transit.
But I fixed it.  What I did find was an error at line 3716 in JcomboBoxUtil.java that trys to insert an object which creates a classexception exception, which is caught every time. I think it just wants any object not an Object. Its above my pay grade.

Reply:
> What I did find was an error at line 3716 in JcomboBoxUtil.java that trys to insert an object which creates a classexception exception

Thanks for finding that.